graspy/embed/base.py

```diff
@@ -158,11 +165,10 @@ def fit_transform(self, graph, y=None):
 
         Returns
         -------
-        out : np.ndarray, shape (n_vertices, n_dimension) OR tuple (len 2)
-            Where both elements have shape (n_vertices, n_dimension)
-            A single np.ndarray represents the latent position of an undirected
-            graph, wheras a tuple represents the left and right latent positions
-            for a directed graph
+        out : np.ndarray OR length 2 tuple of np.ndarray.
+            if undirected then returns single np.ndarray of latent position, shape(n_vertices, n_dimension).
+            if directed, self.concat = True then concatenate latent matrices on axis 1, shape(n_vertices, 2*n_dimension).
```

Comment:
can just say `concat` but want to wrap any variables in double backticks. look through some of the other documentation for examples of how to ref variables